DependencyInjection/Configuration.php

```diff
@@ -213,6 +213,14 @@ public function getConfigTreeBuilder()
                                 continue;
                             }
 
+                            if (isset($template['inherits_containers'])) {
+                                $inherits = $template['inherits_containers'];
+                                if (!isset($templates[$inherits])) {
+                                    throw new InvalidConfigurationException(sprintf('Template "%s" cannot inherits containers from undefined template "%s"', $id, $inherits));
```

Comment:
Docs:
inherits_containers: parent container

I think the reason this had an s in it was so you get **"template X inherists containers from Y"**